backend/internal/controller/adapter/shot_message_test.go

```diff
@@ -75,9 +74,8 @@ func TestUnpackShotMsg(t *testing.T) {
 				Params: rawSlice,
 			}
 			shotJSON, err := json.Marshal(tableShot)
-			if err != nil {
-				assert.FailNow(t, err.Error())
-			}
+
+			assert.Nil(t, err)
```

Comment:
Is this a good idea? In the case of unsuccessful marshalling there is probably no point in performing further tests.

Reply:
Use [`require.Nil`](https://pkg.go.dev/github.com/stretchr/testify@v1.8.0/require#Nil)

Reply:
Done